scripts/meta/writeBotScripts.py

```diff
@@ -0,0 +1,46 @@
+import os
+from lsst.utils import getPackageDir
+
+isrRunnerScript = """
+# This file is part of rubintv_production.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from lsst.rubintv.production.slac import RawProcesser
+from lsst.summit.utils.utils import setupLogging
+
+setupLogging()
+print('Running raw processor for detector {}...')
+isrRunner = RawProcesser(detector={})
+isrRunner.run()
+"""
+
+packageDir = getPackageDir('rubintv_production')
+outputDir = os.path.join(packageDir, 'scripts', 'slac', 'bot')
+if os.path.isdir(outputDir) is False:
+    print(f"Output directory {outputDir} does not exist, creating it")
+    os.makedirs(outputDir)
+
+for detNum in range(205):
+    scriptName = os.path.join(outputDir, f"runIsrRunner_{detNum:03}.py")
+    with open(scriptName, 'w') as f:
+        contents = isrRunnerScript.format(detNum, detNum)
+        f.write(contents)
+        print(f"Wrote runner script for {detNum} to {scriptName}")
```

Comment:
I think a subcommand approach would organize things much better than script-generation, if that's something that still might be viable to change. And `argparse` makes it pretty easy (I would not recommend `click` for this).

```
bin/run-rubintv butler-watcher
bin/run-rubintv isr-runner 018
```
etc.

Reply:
This is complicated, and there are two reasons why I didn't do it. But firstly, yes, you're absolutely right, and we might still go that way. But that means a change in how the kubernetes pods which run these are made, and that's not something I have control over, and try to bug the people who do as little as possible. And secondly, given that I'm still not sure that each won't end up being somewhat hand-crafted (though to what extent I don't know yet, maybe config options per runner, maybe pods doing multiple detectors, but also maybe nothingetc) I thought it best just to keep everything in my control for now.

Also note that it's only the mass-produced pods which are like this, all the one-off ones are still handmade, so it would only be the `isr-runner detNum` ones I think, and I very much plan to keep it that way (lest I need 500 pods!)